core/trino-main/src/main/java/io/trino/operator/HashAggregationOperator.java

```diff
@@ -562,7 +562,7 @@ private Page getGlobalAggregationOutput()
             }
 
             for (AggregatorFactory aggregatorFactory : aggregatorFactories) {
-                aggregatorFactory.createAggregator().evaluate(output.getBlockBuilder(channel));
+                aggregatorFactory.createAggregator(UpdateMemory.NOOP).evaluate(output.getBlockBuilder(channel));
```

Comment:
could you add a comment why inserting `NOOP` is ok here?

Reply:
Done